bloc-did-method.md

```diff
@@ -8,99 +8,165 @@ The `did:bloc` DID method allows groups of independent organizations to share cu
-`did:bloc:[host]:[doc hash]`
+`did:bloc:[host]:[doc ID]`
+
+Where `[host]` is a URL host (eg, `foo.example.net`) and `[doc ID]` is the document ID as used in Sidetree.
 
-Where `[host]` is a URL host (eg, `foo.example.net`) and `[doc hash]` is the document hash as used in Sidetree.
+#### Data Model
 
-#### Consortium Data Model
+##### Consortium Files
 The policies of a consortium, along with information identifying the stakeholders, is stored under the consortium's domain, within `http://[consortium domain]/.well-known/did-bloc`
 
 `.well-known/did-bloc` should have this file hierarchy:
 
     .well-known/did-bloc/
-        consortium/
-            conf.json
-        stakeholders/
+        [domain].json
         history/
-    
-##### The `consortium` directory
-This directory contains the current consortium configuration, according to this particular server.
-
-###### `conf.json`
-The consortium config contains the following:
-- The domain name of the consortium
-- Consortium policy configuration settings
-- A list of stakeholders - containing, for each stakeholder:
-  - The web domain where its configuration can be found
-  - A local hashlink to the cached stakeholder configuration
-- A list of detached JWS signatures, from the stakeholders, on the config file
-- A local hashlink to the previous version of this config file
+
+`[domain].json`:
+This file contains the current consortium configuration, according to this particular server.
+
+`history`: contains the history database.
+
+##### Consortium Config Files
+A consortium config file is a concatenated JSON file:
+The first element is a JSON object containing:
+  - The domain name of the consortium
+  - Consortium policy configuration settings
+  - A list of stakeholders - containing, for each stakeholder:
+    - The web domain where its configuration can be found
+    - The did:bloc DID of the stakeholder, with the associated DID doc in Sidetree on the consortium ledger
+  - The hash of the previous version of this config file
+After that, in order:
+  - A list of detached JWS signatures, from the stakeholders, on the config file
+  - A list of cached copies of the stakeholder config files
 
 ###### Consortium Endorsement Signatures
 Stakeholders endorse the contents of a consortium configuration file by signing the contents.
 
-Verification: The signatures array must be removed from the config file and the config file must be canonicalized (TODO: define how) before the signatures are verified against the file.
+First the file has appended data removed.
+- The signatures array is removed
+- The stakeholder config cache is removed
 
-##### Stakeholder Configurations
-Within `stakeholders/` are files for stakeholder configs.
+Verification: The signatures array must be removed from the config file and the config file must be canonicalized (as done by JWS<!--does jws even canonicalize or does it sign a particular string representation...-->) before the signatures are verified against the file.
```

Comment:
* Canonicalization is the semantic layer's concern, not JWA algorithms.
* TODOs should be left visible in the rendered text